src/test/java/com/cgdp/library/validation/ValidatorTest.java

```diff
@@ -0,0 +1,169 @@
+package com.cgdp.library.validation;
+
+import com.cgdp.library.exceptions.ValidationException;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.catchThrowable;
+
+public class ValidatorTest {
+
+    @Test
+    void should_throw_exception_when_value_is_null() {
+        // given
+        String paramName = "value";
+        String value = null;
+
+        // when
+        Throwable thrownException = catchThrowable(() -> Validator.required(paramName, value));
+
+        // then
+        assertThat(thrownException)
+              .isInstanceOf(ValidationException.class)
+              .hasMessage(String.format("%s must not be null", paramName));
+    }
+
+    @Test
+    void should_throw_exception_when_value_is_not_null() {
+        // given
+        String paramName = "value";
+        String value = "valid";
+
+        // when
+        Throwable thrownException = catchThrowable(() -> Validator.required(paramName, value));
```

Comment:
Why are we trying to catch an exception when we are not expecting an exception?
Test should validate that the value was returned
Same for the other tests